docs/expressions/syntax.md

```diff
@@ -92,6 +93,7 @@ Operator    | Name                       | Syntax      | Associativity | Example
 `xor`       | Logical xor                | `x xor y`   | Left to right | `true xor true`       | `false`
 `=`         | Assignment                 | `x = y`     | Right to left | `a = 5`               | `5`
 `?` `:`     | Conditional expression     | `x ? y : z` | Right to left | `15 > 100 ? 1 : -1`   | `-1`
+`??` `nullish` | Nullish coalescing | `x ?? y` | Right to left | `null ?? 2` | `2`
```

Comment:
No need for `nullish` to appear in this entry; that is a function, not an operator.